Engine/PySideCompat.cpp

```diff
@@ -39,10 +39,8 @@ CLANG_DIAG_OFF(mismatched-tags)
 GCC_DIAG_OFF(unused-parameter)
 GCC_DIAG_OFF(missing-field-initializers)
 #include <basewrapper.h>
-#include <conversions.h>
```

Comment:
use #if/#ifdef, please don't remove

Reply:
The [FreeCAD codebase](https://github.com/FreeCAD/FreeCAD/blob/00bb1d0b58aaba641c271e705d4324ae06a5b117/src/Gui/PythonWrapper.cpp) uses `#if`/`#ifdef` for the Python wrapper, gonna abstain the removals and use the preprocessor instead.

Reply:
OK!
also, I would remommend putting shiboken-generated files and shiboken2 generated files in two separate directories, so that we can check in both in the sources. That would be great

Reply:
Problem is that the `Engine.pro` is gonna need a good amount of tinkering, cannot get any idea to what to do here